desc/compute/_bootstrap.py

```diff
@@ -280,8 +278,8 @@ def j_dot_B_Redl(
     J_dot_B = dnds_term + dTeds_term + dTids_term
 
     # Store all results in the J_dot_B_data dictionary:
-    nu_e_star = nu_e
-    nu_i_star = nu_i
+    nu_e_star = nu_e  # noqa: F841
```

Comment:
if nu_e_star is never used, we can probably get rid of this line and the next line? F841 is local variable assigned but never used

Reply:
it is used, though the `eval` loop below. I refactored this in #601 to remove that part and simplify this, but for now this is fine.